src/libraries/System.Text.Json/src/System/Text/Json/Schema/JsonSchema.cs

```diff
@@ -26,8 +26,8 @@ internal sealed class JsonSchema
         internal const string MinLengthPropertyName = "minLength";
         internal const string MaxLengthPropertyName = "maxLength";
 
-        public static JsonSchema False { get; } = new(false);
-        public static JsonSchema True { get; } = new(true);
+        public static JsonSchema CreateFalseSchema() => new(false);
+        public static JsonSchema CreateTrueSchema() => new(true);
```

Comment:
And we don't just want to use `new(true)` at call sites?

Reply:
The constructor accepting boolean is marked private since boolean schemas are special and do not admit keywords. It's better if the false and true schemas are created explicitly. If I were to redesign this type today, I would have made booleans and objects separate subclasses -- the current arrangement has been the source of most bugs we've seen in `JsonSchemaExporter`.

Reply:
> The constructor accepting boolean is marked private since boolean schemas are special and do not admit keywords.

But aren't these factories now 100% equivalent to the constructor? If it's ok for these factories to exist, I don't understand why the constructor is any more problematic.

Reply:
I don't think `JsonSchema schema = new(false);` adequately conveys that a `false` schema is being created. I would rather assign a distinct factory for that case.